README.md

```diff
@@ -58,6 +60,76 @@ cookiecutter https://github.com/ukgovdatascience/govcookiecutter
 
 Then following the prompts in your terminal to create the project structure.
 
+## Installing pre-commit hooks
+
+This repo uses the Python package [`pre-commit`](https://pre-commit.com) to manage pre-commit hooks. Pre-commit hooks 
+are actions which are run automatically, typically on each commit, to perform some common set of tasks. For example, a
+pre-commit hook might be used to run any code linting automatically, providing any warnings before code is committed,
+ensuring that all of our code adheres to a certain quality standard.
+
+For this repo, we are using `pre-commit` for a number of purposes:
+- Checking for any secrets being committed accidentally;
+- Checking for any large files (over 5MB) being committed; and
+- Cleaning Jupyter notebooks, which means removing all outputs and execution counts.
+
+We have configured `pre-commit` to run automatically on _every commit_. By running on each commit, we ensure
+that `pre-commit` will be able to detect all contraventions and keep our repo in a healthy state.
```

Comment:
Did you mean to add the docs about `pre-commit` to the main repo README, or only the projects which this repo is here to create?

Reply:
Both actually - the main repo for anyone developing the cookiecutter, and the resultant repos that this cookiecutter creates as well!